src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -478,6 +484,13 @@ private ScreenCorner GetScreenCornerForMousePosition(Vector2 mousePosition)
     private void ClampCameraXZPosition()
     {
         TerrainMap map = _session.TerrainMap;
+
+        if (map == null)
+        {
+            Debug.Log("Unable to clamp camera to XZ position. No map data loaded.");
+            return;
+        }
```

Comment:
Same as the minimap remark, needs a comment, and probably should not log anything.